support/investigating-pod-issues.adoc

```diff
@@ -0,0 +1,30 @@
+[id="investigating-pod-issues"]
+= Investigating Pod issues
+include::modules/common-attributes.adoc[]
+:context: investigating-pod-issues
+
+toc::[]
+
+{product-title} {product-version} leverages the Kubernetes concept of a Pod, which is one or more containers deployed together on one host. A Pod is the smallest compute unit that can be defined, deployed and managed on {product-title} {product-version}.
+
+After a Pod is defined, it is assigned to run on a node until its container (or containers) exit, or until it is removed. Depending on policy and exit code, Pods are either removed after exiting or retained so that their logs can be accessed.
+
+When Pod issues arise, determine Pod status. If an explicit Pod failure has occurred, observe the Pod's error state to identify specific image, container or Pod network issues. Focus diagnostic data collection according to the error state. Review Pod event messages, as well as Pod and container log information. Diagnose issues dynamically by accessing running Pods on the command line, or start a debug Pod with root access based on a problematic Pod's deployment configuration.
```

Comment:
This sentence, "When Pod issues arise, determine Pod status." seems a little incomplete, or abrupt, to me. Should it be something like "The first thing to check when Pod issues arise is the Pod's status" ?

Reply:
Yes that sounds much better, thanks. Resolved.